docs/_index.md

```diff
@@ -4,6 +4,501 @@ meta_desc: Provides an overview of the MSSQL Provider for Pulumi.
 layout: overview
```

Comment:
While you are at it, can you change the value from `overview` to `package` in this file *and* in `installation-configuration.md`? I will communicate this more widely for the other Pulumiverse providers, but it was an ask from our content engineering team.

Reply:
Changed it